coralproject/talk
# Notifications: Disable notifications polling when the feature is disabled

## What does this PR do?

- disables the `useLiveNotificationsPolling` hook when `inPageNotifications` is not enabled in the tenant settings

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- enable in page notifications in the admin
- visit stream while logged in, see that polling is happening in the network tab of dev tools
- disable in page notifications in the admin
- visit stream while logged in, see that polling is not happening in the network tab of dev tools

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

Merge into epic branch.
